package.json

```diff
@@ -1,11 +1,13 @@
-{
+{ 
+  "__mplheader": "This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.",
```

Comment:
Do not do this. I recommend using package.json's license field for this very reason.

Reply:
That's not what the license field is for; its purpose is to specify the _name_ (or, rather SPDX ID) of the license being used for the package, not to license the file itself. https://docs.npmjs.com/files/package.json#license

Reply:
In every single open sourced Javascript project, I have never seen a license header in the package.json. We should use the license field for the SPDX ID, and leave out the header.